Makefile

```diff
@@ -92,17 +92,19 @@ compile-native: $(OUTPUT_DIR)/libp2p_nif.so
 
 compile-port: $(OUTPUT_DIR)/libp2p_port
 
+compile: compile-native compile-port $(PROTOBUF_EX_FILES)
```

Comment:
Ah, I thought compile would include Elixir compilation (compile all), and compile-native would include ports (compile everything native).

WDYT?

Reply:
That's a good idea. I'll try it